app/src/main/java/org/mozilla/fenix/components/metrics/GleanMetricsService.kt

```diff
@@ -659,6 +659,12 @@ private val Event.wrapper: EventWrapper<*>?
             { ProgressiveWebApp.backgroundKeys.valueOf(it) }
         )
 
+        Event.SyncedTabOpened -> EventWrapper<NoExtraKeys>(
+            { Metrics.syncedTabsOpenedCount.add(1) }
```

Comment:
> A counter that indicates how many synced tabs a user has opened

The description for this event reads like we want to count the number of times a synced tab _item_ is clicked and not the synced tabs UI is opened. Is that right?

If so, we probably want to add this counter to where the [tab was clicked](https://github.com/mozilla-mobile/fenix/blob/77f061c362dfb64de563eee60ec8c694c8cf290e/app/src/main/java/org/mozilla/fenix/sync/SyncedTabsLayout.kt#L120).

Reply:
We spoke offline and clarified how metrics are recorded.